guides/source/active_storage_overview.md

```diff
@@ -572,6 +572,39 @@ You can bypass the content type inference from the data by passing in
 If you don’t provide a content type and Active Storage can’t determine the
 file’s content type automatically, it defaults to application/octet-stream.
 
+### Replace vs Add Attachments
+
+By default in Rails, attaching files to a `has_many_attached` association will replace
+any existing attachments. To instead add new attachments while retaining existing
+ones, set `Rails.application.config.replace_on_assign_to_many` to false.
+
+Alternatively, you can add hidden form fields with the [`signed_id`][ActiveStorage::Blob#signed_id]
+of each attached file:
```

Comment:
We try to avoid using "you" in wording for Rails docs: https://edgeguides.rubyonrails.org/api_documentation_guidelines.html#wording 😄 

```suggestion
Alternatively, use hidden form fields with the [`signed_id`][ActiveStorage::Blob#signed_id]
of each attached file:
```